project_task_knowledge/views/project_task_views.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_project_task_knowledge_form" model="ir.ui.view">
+        <field name="name">Project Task (Knowledge)</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2" />
+        <field name="groups_id" eval="[(4, ref('knowledge.group_document_user'))]"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='project_id']" position="after">
+                <field name="document_page_ids" mode="tree">
+                    <field name="name" />
+                    <field name="parent_id" />
+                </field>
```

Comment:
Is it possible rendering a button box with "X wiki pages", as `document_page_project` does, instead of tree field?

https://github.com/OCA/knowledge/blob/14.0/document_page_project/views/project_project_views.xml#L32